pkg/util/interceptor.go

```diff
@@ -39,3 +40,22 @@ func (s timeoutInterceptor) WrapStreamingHandler(next connect.StreamingHandlerFu
 		return next(ctx, shc)
 	})
 }
+
+// LogRequest logs the request parameters.
+// It logs all kinds of requests.
+func NewLogInterceptor(logger log.Logger) connect.UnaryInterceptorFunc {
```

Comment:
I prefer this approach.

Reply:
But we missing some information here. For example method, response, etc.
At the same time, I guess this will overlap with the logging middleware.

Reply:
Yeah I guess it's fine for now. Can you get the tenantID and the trace ID and the duration too ?

This is mostly to discover parameters of slow queries.

Reply:
Added. Thanks!